README.md

```diff
@@ -1,85 +1,74 @@
- * Cryptographic connection entanglement - allowing linking of data points only when specific cryptographic rules are satisfied
- * Trust minimization through decentralization - utilizing a decentralized p2p overlay network for data exchange and the Ethereum blockchain in the consensus layer
- 
- 
-Therefore the key development principles of OriginTrail ecosystem are:
+The ODN as a network holds a growing Decentralized Knowledge Graph (DKG) with the following characteristics:
+* Linked data first structure - the graph, enabling connections between data points from all published datasets on the network, conformant with Semantic Web technologies such as RDF and JSON-LD
+* Schema flexibility - enabling the mapping of virtually any data model, preferably structured according to relevant standards (such as GS1 EPCIS and CBV) and recommendations (W3C Web of Things, Verifiable Credentials, PROV, etc.) for machine readability
+* Identity verification - enabling the utilization of novel identity frameworks such as Self-sovereign identity, in conjunction with industry-specific identity frameworks (such as GS1 GTIN, GIAI, GRAI and other identification schemes)
+* Efficient cryptographic integrity verification of subgraphs, using associated dataset graph fingerprints, computed as Merkle roots of the input datasets
+* Cryptographic connection entanglement - allowing linking of data points only when specific cryptographic rules are satisfied
+* Trust minimization through decentralization - utilizing a decentralized p2p overlay network for data exchange and the Ethereum blockchain in the consensus layer
 
- * Connection-first approach - providing ways to connect the world's data into a global, decentralized knowledge graph
- * Technological neutrality - avoiding technological lock-ins and striving towards agnosticism where possible
- * Decentralization - designing, implementing and utilizing that are not based on trusted-third parties or centralized entities
- * Privacy-by-Design approach - according to the [7 Foundational Principles of Privacy by Design](https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf)
- * Development transparency - towards the OriginTrail Ecosystem community of developers, node holders and businesses
- * Open Source Development - according to [Open Source Software](https://en.wikipedia.org/wiki/Open-source_model) principles
- 
- 
-Contribution
-============
+Therefore the key development principles of OriginTrail ecosystem are:
+* Connection-first approach - providing ways to connect the world's data into a global, decentralized knowledge graph
+* Technological neutrality - avoiding technological lock-ins and striving towards agnosticism where possible
+* Decentralization - designing, implementing and utilizing that are not based on trusted-third parties or centralized entities
+* Privacy-by-Design approach - according to the 7 Foundational Principles of Privacy by Design
```

Comment:
Previously this was link to: (https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf), did we remove it on purpose?